scijava/scijava-ops/src/main/java/org/scijava/ops/OpEnvironment.java

```diff
@@ -139,4 +147,16 @@ default OpBuilder op(final String opName) {
 	 * @param name
 	 */
 	void register(OpInfo info, String name);
+
+	/**
+	 * Sets the {@link Hints} for the {@link OpEnvironment}. Every Call to
+	 * {@link #op} that <b>does not</b> pass a {@link Hints} will <b>copy</b> the
+	 * Hints passed here (to prevent multiple Op calls from accessing/changing the
+	 * same {@link Hints}). In the case that <b>no {@link Hints} have been
+	 * given</b> to the Environment before {@code op} is called, the
+	 * implementation will create a suitable stand-in.
+	 * 
+	 * @param hints
+	 */
+	void setHints(Hints hints);
```

Comment:
TODO: add a disclaimer that to guarantee the `Hints` that are used during an op call, you should include the `Hints` in the call instead of at this environment level.

Reply:
Done!